File: app/src/main/java/com/example/msdc/api/EpisodeResult.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class EpisodeResult {
    @SerializedName("air_date")
    String airDate;

    @SerializedName("episode_number")
    int episodeNumber;

    @SerializedName("id")
    int id;

    @SerializedName("name")
    String name;

    @SerializedName("overview")
    String overview;

    @SerializedName("runtime")
    int runtime;

    @SerializedName("season_number")
    int seasonNumber;

    @SerializedName("still_path")
    String stillPath;

    public String getAirDate() {
        return airDate;
    }

    public int getEpisodeNumber() {
        return episodeNumber;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getOverview() {
        return overview;
    }

    public int getRuntime() {
        return runtime;
    }

    public int getSeasonNumber() {
        return seasonNumber;
    }

    public String getStillPath() {
        return stillPath;
    }
}

File: app/src/main/java/com/example/msdc/api/EpisodeResult.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class EpisodeResult (
    @field:SerializedName("id") val id: Int? = 0,
    @field:SerializedName("air_date") val airDate: String? = null,
    @field:SerializedName("episode_number") val episodeNumber: Int? = 0,
    @field:SerializedName("name") val name: String? = null,
    @field:SerializedName("overview") val overview: String? = null,
    @field:SerializedName("runtime") val runtime: Int? = 0,
    @field:SerializedName("season_number") val seasonNumber: Int? = 0,
    @field:SerializedName("still_path") val stillPath: String? = null
) : Parcelable